app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    cmd, source_path, destination_path = command.split(" ")
```

Comment:
Can you check:
command == "mv"
It it will raise an error? As for me yes. You need validate for command.split() before this line

Reply:
> Can you check: command == "mv" It it will raise an error? As for me yes. You need validate for command.split() before this line
should i try:
```
    if isinstance(command, str) and len(command.split()) == 3:
        cmd, source_path, destination_path = command.split()
```
?